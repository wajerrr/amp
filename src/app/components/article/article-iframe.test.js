import React from 'react';
import renderer from 'react-test-renderer';
import ArticleIframe from './article-iframe';

describe('ArticleIframe', () => {
  it('should match snapshot', () => {
    const tree = renderer
      .create(
        <ArticleIframe
          src="https://infographics.economist.com/2017/ukelmap-2017/"
          height={100}
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
