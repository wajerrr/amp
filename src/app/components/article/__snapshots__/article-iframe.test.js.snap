// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleIframe should match snapshot 1`] = `
<amp-iframe
  frameborder="0"
  height={100}
  layout="fixed-height"
  sandbox="allow-scripts allow-same-origin"
  src="https://infographics.economist.com/2017/ukelmap-2017/"
>
   
  <amp-img
    height={100}
    layout="fixed-height"
    placeholder={true}
    src="https://www.economist.com/assets/the-economist-logo.svg"
  />
</amp-iframe>
`;
